esp/templates/inclusion/program/class_teacher_list_row.html

```diff
@@ -1,4 +1,32 @@
-{# expects program, class, and crmi (ClassRegModuleInfo) #}
+{# expects program, class, can_req_cancel, and crmi (ClassRegModuleInfo) #}
+
+<style>
+.clsmiddle {
+    vertical-align: middle;
+}
+
+.cancelbutton {
+    color: #000000;
+    font-family: Arial,helvetica,sans-serif;
+    font-size: 11px;
+    font-weight: bold;
+    border: #cccccc 1px solid;
+    background-color: #cccccc;
+    cursor: default;
+    padding: 2px 5px 2px 5px;
+    text-decoration: none;
+    margin: 0px;
+    line-height: normal;
+    cursor: pointer;
+    box-shadow: none;
+    transition: border linear 0.2s, box-shadow linear 0.2s;
+    border-radius: 0;
+}
+
+.teachbutton {
```

Comment:
What's this change?

Reply:
For some reason, the buttons had asymmetric margins, which made them not centered vertically.